Installing @defencedigital/react-component-library or @defencedigital/icon-library gets stuck in a loop

Running:

```shell
npx create-react-app@latest --template defencedigital
```

Currently results in an infinite loop of messages stating:

```
npm WARN Could not resolve dependency:
npm WARN peer react@"15.x.x && 16.x.x" from the root project
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: react-children-utilities@1.3.3
npm WARN Found: peer react@"15.x.x && 16.x.x" from the root project
```

--- a/packages/icon-library/.svgrrc.js
+++ b/packages/icon-library/.svgrrc.js
@@ -1,13 +1,12 @@
 const template = ({ componentName, exports, interfaces, jsx }, { tpl }) => {
   return tpl`
 import React from 'react'
-import { SVGUniqueID } from 'react-svg-unique-id'
 import { SVGIconProps } from '../types'
 
 ${interfaces}
 
 const ${componentName} = ({ size = 16, ...props }: SVGIconProps) => (
-  <SVGUniqueID>{${jsx}}</SVGUniqueID>
+  ${jsx}
 );
 
 ${exports}
@@ -20,6 +19,11 @@ module.exports = {
   prettierConfig: {
     ...require('./prettier.config.js'),
     parser: 'typescript',
+  },
+  jsx: {
+    babelConfig: {
+      plugins: ['react-inline-svg-unique-id'],
+    },
   },
   svgProps: {
     width: '{size}',

--- a/packages/icon-library/package.json
+++ b/packages/icon-library/package.json
@@ -30,7 +30,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.16.7",
-    "react-svg-unique-id": "^1.3.3"
+    "@inline-svg-unique-id/react": "^1.2.2"
   },
   "devDependencies": {
     "@babel/cli": "^7.8.3",
@@ -43,6 +43,7 @@
     "@svgr/cli": "^6.1.2",
     "@types/react": "^17.0.38",
     "babel-loader": "^8.0.6",
+    "babel-plugin-react-inline-svg-unique-id": "^1.3.1",
     "clean-webpack-plugin": "^4.0.0",
     "npm-run-all": "^4.1.5",
     "react": "^17.0.2",

